packages/augur-core/source/contracts/reporting/Universe.sol

```diff
@@ -93,11 +93,24 @@ contract Universe is IUniverse {
         daiVat = IDaiVat(augur.lookup("DaiVat"));
         daiPot = IDaiPot(augur.lookup("DaiPot"));
         daiJoin = IDaiJoin(augur.lookup("DaiJoin"));
+        assertContractsNotZero();
         daiVat.hope(address(daiPot));
         daiVat.hope(address(daiJoin));
         cash.approve(address(daiJoin), 2 ** 256 - 1);
     }
 
+    function assertContractsNotZero() private view {
+        require(marketFactory != IMarketFactory(0));
+        require(disputeWindowFactory != IDisputeWindowFactory(0));
+        require(shareToken != IShareToken(0));
+        require(repPriceOracle != IRepPriceOracle(0));
+        require(formulas != IFormulas(0));
+        require(cash != ICash(0));
+        require(daiVat != IDaiVat(0));
+        require(daiPot != IDaiPot(0));
+        require(daiJoin != IDaiJoin(0));
+    }
```

Comment:
Out of curiosity, why did you break this out into a separate function?  It feels like splitting it like this is going to increase the chances of missing/forgetting one vs something like:
```solidity
marketFactory = IMarketFactory(augur.lookup("MarketFactory"))
require(marketFactory != IMarketFactory(0);
```
Also, if you wanted to save a little gas by using a local variable:
```solidity
address _temp;
_temp = augur.lookup("MarketFactory");
require(_temp != address(0));
marketFactory = IMarketFactory(_temp);
_temp = augur.lookup("DisputeWindowFactory");
require(_temp != address(0));
disputeWindowFactory = IDisputeWindowFactory(_temp);
...
```

Reply:
Thought this was more readable.